### ISSUE TITLE
Windows binaries don't run on Windows 7

### ISSUE THREAD

**Issue by fgm** [fyne-io/fyne#1923]
### Describe the bug:

Binaries built on macOS run on Windows 10 Pro, but not on Windows 7 Home Edition Premium.

### To Reproduce:
Steps to reproduce the behaviour:

1. On macOS, copy the First App from https://developer.fyne.io/started/firstapp, procure an image to use as an icon, say `someimage.png`
2. Install the XCode CLI development tools with `xcode-select --install`
3. Install MinGW with `brew mingw-w64`
4. Build the app with `CGO_ENABLED=1 CC=x86_64-w64-mingw32-gcc fyne package -os windows -icon someimage.png`
5. Check the binary with `file firstapp.exe`, it shows `PE32+ executable (GUI) x86-64, for MS Windows`.
6. Transfer the binary to a W10 PRo and a W7 machine.
7. It runs on the former, and is rejected on the latter, with an error about it not being for the proper windows version.
8. Alternatively, trying to build a 32-bit version with CGO_ENABLED=1 CC=i686-w64-mingw32-gcc fyne package -os windows -icon someimage.png` fails with `cc1: sorry, unimplemented: 64-bit mode not compiled in`.

### Device (please complete the following information):
 - **OS:** Dev: macOS, Target: Windows
 - **Version:** Dev: Catalina, Target: Windows 10 Pro, Windows 7 Home Edition Premium
 - **Go version:** 1.15.8
 - **Fyne version:** fyne.io/fyne/v2 v2.0.0

**Comment by tehsphinx** [comment]
Is that something fyne should/can support or are there major steps involved to get this working? This is kind of a "deal breaker" for my use case.

**Comment by andydotxyz** [comment]
Yes we should support Windows 7. I don't think that any of the core devs have access to a machine with that version for testing.
I suspect it is a "simple" compile parameter in `fyne package` but I personally don't know Windows well enough to guess which.

**Comment by fgm** [comment]
FWIW, if you send me binaries, I can try them on mine.

**Comment by andydotxyz** [comment]
As of Fyne v2.8.0 we don't support Windows 7, sorry. Go dropped it years ago and we can't keep holding on to old releases. In v2.8.0 we require Go 1.22 or higher.
